### PR TITLE
Build: upgrade controller-gen to 0.16.3

### PR DESCRIPTION
**Pull request by benashz**
This PR should unblock #943

### CODE REVIEW

File: chart/crds/secrets.hashicorp.com_vaultauths.yaml [hunk 1]
```diff
@@ -336,16 +329,8 @@ spec:
...
-                  description: "Condition contains details for one aspect of the current
-                    state of this API Resource.\n---\nThis struct is intended for
-                    direct use as an array at the field path .status.conditions.  For
-                    example,\n\n\n\ttype FooStatus struct{\n\t    // Represents the
-                    observations of a foo's current state.\n\t    // Known .status.conditions.type
-                    are: \"Available\", \"Progressing\", and \"Degraded\"\n\t    //
-                    +patchMergeKey=type\n\t    // +patchStrategy=merge\n\t    // +listType=map\n\t
-                    \   // +listMapKey=type\n\t    Conditions []metav1.Condition `json:\"conditions,omitempty\"
-                    patchStrategy:\"merge\" patchMergeKey:\"type\" protobuf:\"bytes,1,rep,name=conditions\"`\n\n\n\t
-                    \   // other fields\n\t}"
```
Comment (tvoran) [inline chart/crds/secrets.hashicorp.com_vaultauths.yaml:348]: Any concerns with this and the other descriptions in the CRD definitions being shortened?

**Review by tvoran — APPROVED** [review]
Are the chart upgrade test failures expected?

File: chart/crds/secrets.hashicorp.com_vaultauths.yaml (hunk 1 above)
Comment (benashz) [reply chart/crds/secrets.hashicorp.com_vaultauths.yaml:348]: I am not sure why upstream dropped this. It seems to be the default now.

**Comment by benashz** [comment]
> Are the chart upgrade test failures expected?

No, totally unexpected. Taking a closer look.

**Pull request merged**